### ISSUE TITLE
Install instructions don't seem to work

### ISSUE THREAD

**Issue by rugene76** [fwartner/ha-openwakeword-installer#5]
I've got HACS installed, I download the wakework installer repository  but that's as far as I can get.

Maybe I'm completely missing the point, but the installer never shows up in the settings-->integration-->"add integration" and the instructions refer to "openwakeword" not "wakeword_installer". Yes, I rebooted.

what am I missing?

for now I just manually copying the desired wakeword tflite files by hand...

**Comment by fwartner** [comment]
Should be fixed now :)

**Comment by rfam13** [comment]
The integration shows up as "wakeword installer".  I have tried to use it multiple times but when I input the repo URL nothing happens.  Just shows a blank integration without entities.  Not sure if it is supposed to show anything?

**Comment by rugene76** [comment]
For some reason after a few reboots and upgrading to the latest HA core, the wakeword_installer showed up.

As rfam13, I've configured it to point to https://github.com/fwartner/home-assistant-wakewords-collection.  I'm also pointing at "en" on the second line of the wakeword_installer config dialog.

I've stored  my wakewords (tflite) in "share/openwakeword" but I don't see any additional files populated there beyond what I did manually. Where should the wakewords be stored?
thanks
